packages/editor/src/components/post-author/constants.js

```diff
@@ -0,0 +1,6 @@
+export const AUTHORS_QUERY = {
+	who: 'authors',
+	per_page: 50,
+	_fields: 'id,name',
+	context: 'view',
```

Comment:
Maybe add a comment explaining that the context is "view" in order to allow non-admin roles perform the request.